### ISSUE TITLE
Restart the project with proper practices and updated technologies.

### ISSUE THREAD

**Issue by pkspyder007** [pkspyder007/Excalibur#15]
The project will be written from ground up from frontend to backend to provide seemless user experience and ability to manage the codebase easily and make it easy for new developer to hop in.

**Comment by pkspyder007** [comment]
@telegrambotdev  would to like to work on the project?

**Comment by rsoorajs** [comment]
> @telegrambotdev would to like to work on the project?

yup.i will help you which i can.so where are we stuck in the current code ?

**Comment by pkspyder007** [comment]
@telegrambotdev so we are not having proper front end and backend connection by that i mean every route and redirect call is hard coded , a lot of callbacks are used and the exceptions are not handled. 
So if we start the project again with simple goals it Will be easy to maintain. 
I'll make a file containing all the details for the steps.
